docs/ref/contrib/contenttypes.txt

```diff
@@ -349,6 +349,10 @@ normal field object, these examples will *not* work::
 Likewise, :class:`~django.contrib.contenttypes.fields.GenericForeignKey`\s
 does not appear in :class:`~django.forms.ModelForm`\s.
 
+Keep in mind that :class:`~django.contrib.contenttypes.fields.GenericForeignKey`
```

Comment:
To me, "Keep in mind" suggests that this has already been explained. It would be nice to give an example of how this situation arises.

Reply:
Thanks @timgraham I will add an example.